### ISSUE TITLE
MPI issues

### ISSUE THREAD

**Issue by chomikzimon** [SSAGESproject/SSAGES#25]
Hi! 
Let me start by thanking you for your code!

We have implemented a new CV that calculates the `cvvalue` in parallel. That was needed as the algorithm considers the contribution of every atom in the system. We have a local version of SSAGES that contains the new piece of code. When everything works, we would like to share it with you. Unfortunately, we noticed that when running FFS with LAMMPS the “dumpfiles” are written by SSAGES serially, so each processor overwrites the output. So, e.g., in a file "l0-n0.dat" we only have information about the atoms handled by one core, and we receive the following error ```error, could not locate atomID 1 from dumpfile```. We would like to use SSAGES for a large MD calculations. Therefore, parallelisation is important to us. Have we missed anything? 

We would really appreciate your help.

**Comment by mquevill** [comment]
You are correct that multiple MPI processes are trying to read/write at the same time, which corrupts the dumpfiles. Because of this and a few other quirks, the current implementation of FFS is limited to just 1 MPI process per walker. We hope to fix the full parallelization capabilities of FFS soon, but we do not have an explicit timeline for when that will be. See also https://github.com/SSAGESproject/SSAGES/issues/10#issuecomment-498678419.

As a current workaround to accelerate simulations, you could try utilizing OpenMP threads with the USER-OMP package. It is only the MPI processes per walker that are limited.

**Comment by chomikzimon** [comment]
Thank you for your response.

**Comment by mquevill** [comment]
SSAGES v0.9.3 will now throw an error if the user tries to use more than 1 MPI process per walker. While we would like to get full MPI support for FFS, this should prevent the generation of corrupted data files.
